.editorconfig

```diff
@@ -23,9 +24,14 @@ indent_size = 2
 [webapp/i18n/**.json]
 indent_size = 2
 
-[Makefile]
+[Makefile,*.mk]
 indent_style = tab
 
 [*.scss]
```

Comment:
This can be removed, as there are no such files in the repo any longer.

Reply:
There's actually still a few in the build directory

Reply:
Oh, you meant the *.scss files. I'll take another pass to remove any webapp-only files